bare-metal/container-linux/kubernetes/profiles.tf

```diff
@@ -114,6 +114,30 @@ resource "matchbox_profile" "flatcar-install" {
   container_linux_config = "${element(data.template_file.container-linux-install-configs.*.rendered, count.index)}"
 }
 
+// Flatcar Linux Install profile (from matchbox /assets cache)
+// Note: Admin must have downloaded os_version into matchbox assets.
```

Comment:
into matchbox assets/flatcar

Reply:
Changed, also in the comment above for the cached container linux install.